projects/vdk-plugins/airflow-provider-vdk/vdk_provider/sensors/vdk.py

```diff
@@ -45,4 +50,30 @@ def poke(self, context: Dict[Any, Any]) -> bool:
         :param context: Airflow context passed through the DAG
         :return: True if some job status condition is met; False otherwise
         """
-        pass
+        vdk_hook = VDKHook(self.conn_id, self.job_name, self.team_name)
+
+        job_execution = vdk_hook.get_job_execution_status(self.job_execution_id)
+        job_status = job_execution.status
+
+        log.info(
+            f"Current status of job execution {self.job_execution_id} is: {job_status}."
+        )
+
+        if job_status == "succeeded":
```

Comment:
I think we should print the execution log event if the job fails.

Reply:
Makes sense.